cluster/aks/util.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
FYI cluster/ is considered deprecated and we're moving towards eliminating it (there are various open issues). I expect kubemark to be moved at some point with the other bits worth keeping, but I don't generally recommend expanding upon the rest.

the concept of "detect master" is also something we should remove long term :/

Reply:
kubemark itself isn’t deprecated, right? I’m trying to avoid adding to logic to cluster/ as much as possible. Really I want the minimal change set that enables us to run the kubemark/cluster loader tests.

We dropped some simplistic logic to build a cluster, but really I think we would build a cluster beforehand and Set USE_EXISTING=true. The kubetest changes have that logic, as an example.

Reply:
kubemark is not, along with the conformance image and one or two things I'm not thinking of that aren't cluster lifecycle things... 
*most* of the scripting is though, it has to be maintained for our CI, but longterm the goal is to move it out and / or remove it.